src/WordPress/Blueprints/Runner/Step/DownloadWordPressStepRunner.php

```diff
@@ -7,6 +7,7 @@
 
 class DownloadWordPressStepRunner extends InstallAssetStepRunner {
 
+    //TODO why is this download and not install?
```

Comment:
Because the developer may not want to run the installation wizard automatically.